test/unit/processing/delta/calculate-delta.test.js

```diff
@@ -87,9 +87,10 @@ describe('calculate delta', () => {
         value: -50
       }]
     }]
-    const deltaPaymentRequest = calculateDelta(paymentRequest, previousPaymentRequests)
-    const updatedPaymentRequests = deltaPaymentRequest.completedPaymentRequests
-    expect(updatedPaymentRequests[0].ledger).toBe(AR)
+    const calDeltaPaymentRequest = calculateDelta(paymentRequest, previousPaymentRequests)
+    const { deltaPaymentRequest, completedPaymentRequests } = calDeltaPaymentRequest
+    expect(completedPaymentRequests[0].ledger).toBe(AR)
+    expect(deltaPaymentRequest.netValue).toBe(80)
```

Comment:
I would revert this test back to how it was and create new tests with the description of testing that delta payment request's net value is as expected (just as the payment request comes in). I would create tests for each of the case outlined in this test file already:
- top up with AP
- top up with AR
- recovery as AP
- recovery as AR
- ledger split as AR
- ledger split as AP
- zero value split